src/inputgroup/index.css

```diff
@@ -12,12 +12,20 @@
 /* topdoc
 {{ inputgroup/datepicker-quiet.yml }}
 */
+/* topdoc
+{{ inputgroup/datepicker-range.yml }}
+*/
+/* topdoc
+{{ inputgroup/datepicker-range-quiet.yml }}
+*/
 
 :root {
   /* Todo: move to DNA */
   --spectrum-combobox-quiet-fieldbutton-border-radius: 0;
   --spectrum-combobox-field-border-width-right: 0;
   --spectrum-combobox-quiet-fieldbutton-padding-right: 0;
+  --spectrum-datepicker-input-width: calc(var(--spectrum-global-dimension-size-1600) -  2 * var(--spectrum-padding));
+  --spectrum-datepicker-datetime-input-width: calc(var(--spectrum-datepicker-input-width) + var(--spectrum-global-dimension-size-700) - 0.5em);
```

Comment:
Perhaps @NateBaldwinDesign can provide guidance on the right width to use here? Also, I assume this will work fine with dates using a period or `-` separator instead of `/` (the date won't overlap the em dash)?